shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -762,6 +770,14 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
      */
     @Override
     public boolean performAction(int virtualViewId, int accessibilityAction, @Nullable Bundle arguments) {
+        if (virtualViewId >= MIN_ENGINE_GENERATED_NODE_ID) {
```

Comment:
Do you enforce in the framework that we wrap the id generation at MIN_ENGINE_GENERATED_NODE_ID? If not, please do.